### PR TITLE
Add installation instructions how to install kn-plugin-event via Homebrew

### PR DESCRIPTION
**Pull request by creydr**
# Changes

- :gift: Installation instructions for Homebrew

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +152,14 @@ Run it
 $ kn event -h
 ```
 
+### Install using Homebrew
```
Comment (dsimansk) [inline README.md:155]: I'd probably merge with "Install to work with `kn` CLI" and have something like "Install to work with `kn` CLI from Homebrew". And pointing to kn homebrew instructions first, or just adding the install cmd in a code block.

https://knative.dev/docs/client/install-kn/#install-the-knative-cli

File: README.md [hunk 2]
```diff
@@ -114,6 +114,10 @@ $ kn event send \
 
 ## Install
 
+There are multiple ways to install the `kn-plugin-event`.
+
+### Install using a binary
+
```
Comment (dsimansk) [inline README.md:120]: Adding a link to knative.dev docs? https://knative.dev/docs/client/kn-plugins/#manually-install-a-plugin

**Comment by dsimansk** [comment]
@creydr gentle ping. Fancy going back to this PR? :)

File: README.md (hunk 1 above)
Comment (creydr) [reply README.md:155]: The reason why I added it as a separate heading on the same level as "Install using a binary" was, that you don't need to download the binary manually when using homebrew :shrug:

File: README.md (hunk 2 above)
Comment (dsimansk) [reply README.md:120]: Well of course I've forgot my idea here. I'll check the whole thing once again. In general I'd point to the content on knative.dev when existing already.

**Pull request closed without merging**